qtism/data/state/OutcomeDeclaration.php

```diff
@@ -32,317 +32,366 @@
+     * Systems may exclude outcomes from result reports on the basis of their
+     * declared view if appropriate. Where more than one class of user should
+     * be able to view an outcome variable the view attribute should contain a
+     * comma delimited list.
+     *
+     * @var Viewcollection
+     * @qtism-bean-property
+     */
+    private $views;
+
+    /**
+     * From IMS QTI:
+     *
+     * A human interpretation of the variable's value.
+     *
+     * @var string
+     * @qtism-bean-property
+     */
+    private $interpretation = '';
+
+    /**
+     * From IMS QTI:
+     *
+     * An optional link (URI) to an extended interpretation of the outcome variable's value.
+     *
+     * @var string
+     * @qtism-bean-property
+     */
+    private $longInterpretation = '';
+
+    /**
+     * Normal Maximum. If false, means it was not specified.
+     *
+     * From IMS QTI:
+     *
+     * The normalMaximum attribute optionally defines the maximum magnitude of numeric outcome
+     * variables, it must be a positive value. If given, the outcome's value can be divided
+     * by normalMaximum and then truncated (if necessary) to obtain a normalized score in
+     * the range [-1.0,1.0]. normalMaximum has no affect on responseProcessing or the values
+     * that the outcome variable itself can take.
+     *
+     * @var boolean|number
+     * @qtism-bean-property
+     */
+    private $normalMaximum = false;
+
+    /**
+     * Normal Minimum. If false, means it was not specified.
+     *
+     * From IMS QTI:
+     *
+     * The normalMinimum attribute optionally defines the minimum value of numeric outcome
+     * variables, it may be negative.
+     *
+     * @var boolean|number
+     * @qtism-bean-property
+     */
+    private $normalMinimum = false;
+
+    /**
+     * Mastery Value. If false, means it was not specified.
+     *
+     * From IMS QTI:
+     *
+     * The masteryValue attribute optionally defines a value for numeric outcome variables
+     * above which the aspect being measured is considered to have been mastered by the candidate.
+     *
+     * @var boolean|number
+     * @qtism-bean-property
+     */
+    private $masteryValue = false;
+
+    /**
+     * The lookup table. See LookupTable for more information.
+     *
+     * @var LookupTable
+     * @qtism-bean-property
+     */
+    private $lookupTable = null;
+
+    /**
+     * The externalScored attribute is determining custom way to score item
+     * @var integer|null
+     */
+    private $externalScored = null;
+
+    /**
+     * Create a new instanceof OutcomeDeclaration.
+     *
+     * @param string $identifier A QTI identifier.
+     * @param int $baseType A value from the BaseType enumeration.
+     * @param int $cardinality A value from the Cardinality enumeration.
+     * @param DefaultValue $defaultValue A DefaultValue object.
+     * @param integer|bool $externalScored A ExternalScore object.
```

Comment:
`$externalScored` is either an `integer` or `null`, not `integer` or `bool`.
Also do prefer `int` to `integer`.